### PR TITLE
Feat: Create doc relationship input

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

Creates doc relationship input

## Test Plan

Manual

## Related PRs and Issues

- #327 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/[[page]]/+page.svelte [hunk 1]
```diff
@@ -103,6 +134,21 @@
...
+        id="related"
+        label="Related Collection"
+        bind:search={selectSearch}
+        bind:value={selectValue}
+        required
+        placeholder="Search by ID"
+        options={filteredOptions}
+        let:option={o}>
+        <div class="u-flex u-gap-16">
+            <span style:color="hsl(var(--color-neutral-70))">{o.label}</span>
```
Comment (ArmanNik) [inline src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/[[page]]/+page.svelte:147]: Btw @TGlide I think you can find some already stylized examples at the bottom of the page https://ui-kitchensink-git-style-level-9-appwrite.vercel.app/user.html :)

Comment (TGlide) [reply src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/[[page]]/+page.svelte:147]: Added custom styling when value is selected, thanks!

File: src/lib/elements/forms/inputSelectSearch.svelte [hunk 2]
```diff
@@ -36,16 +38,13 @@
...
         hasFocus = true;
-        clearTimeout(timer);
-        timer = setTimeout(() => {
-            const target = event.target as HTMLInputElement;
-            search = target.value;
-        }, debounce);
     }
 
     function handleKeydown(event: KeyboardEvent) {
+        console.log('handle key down');
```
Comment (TorstenDittmann) [inline src/lib/elements/forms/inputSelectSearch.svelte:46]: leftover

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/[[page]]/+page.svelte [hunk 3]
```diff
@@ -86,6 +87,36 @@
...
+            value: '1',
+            label: 'first-ec68ac3515e4bc06',
+            name: 'Option 1'
+        },
+        {
+            value: '2',
+            label: 'second-ec68ac3521e4bc06',
+            name: 'Option 2'
+        }
+    ];
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/[[page]]/+page.svelte:108]: leftover?

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

File: src/lib/elements/forms/inputSelectSearch.svelte (hunk 2 above)
Comment (TGlide) [reply src/lib/elements/forms/inputSelectSearch.svelte:46]: oops

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/[[page]]/+page.svelte [hunk 4]
```diff
@@ -86,6 +87,36 @@
...
+
+    let selectSearch = '';
+    let selectValue = '';
+
+    $: filteredOptions = (function filter() {
+        console.log('filtering');
```
[the remark this answers is not in the document]
Comment (TGlide) [reply src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/[[page]]/+page.svelte:114]: oops

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**